pkg/trace/writer/sender.go

```diff
@@ -46,14 +46,16 @@ func newSenders(cfg *config.AgentConfig, r eventRecorder, path string, climit, q
 			os.Exit(1)
 		}
 		senders[i] = newSender(&senderConfig{
```

Comment:
Maybe we shouldn't consider evenly splitting the max conns on line 39 for MRF endpoints as this could limit the number of connections we use when MRF is not active (which I assume would be the default state?)

Reply:
well spotted! I will fix this, we should definitely keep max_conns for the expected senders only (MRF's default state is indeed not being active)